### PR TITLE
[OrderState] avoid fill state crash

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_trading/personal_data/orders/states/fill_order_state.py [hunk 1]
```diff
@@ -81,6 +81,8 @@ async def terminate(self):
         `force_close = True` because we know that the order is successfully filled.
         """
         try:
+            self.ensure_not_cleared(self.order)
+
             self.log_event_message(enums.StatesMessages.FILLED)
 
             # call filling actions
```
Comment (Herklos) [inline octobot_trading/personal_data/orders/states/fill_order_state.py:88]: Isn't it cleared during `on_fill_actions` ?

Comment (GuillaumeDSM) [reply octobot_trading/personal_data/orders/states/fill_order_state.py:88]: It's cleared after the cancel order state. The issue is when a cleared order calls `self.order.generate_executed_time()` (a few lines bellow), it crashes

Comment (Herklos) [reply octobot_trading/personal_data/orders/states/fill_order_state.py:88]: Are we sure the previous `self.ensure_not_cleared(self.order)` is not necessary now?

Comment (GuillaumeDSM) [reply octobot_trading/personal_data/orders/states/fill_order_state.py:88]: Right, in case it's cleared in on_fill_action, i'll add a check

**Review by Herklos — APPROVED** [review]

**Pull request merged**